CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ Released: TBD
   names of the form `npm:<package-name>/<filename>` to load library rules from
   an NPM package that is installed relative to the previous non-npm file name,
   or to the current working directory if this is the first file name.
+- [#456](https://github.com/peggyjs/peggy/pull/456) BREAKING: Allow imports from
+  external compiled grammars inside a source grammar, using
+  `import {rule} from "external.js"`.  This will need require explicit support
```

Comment:
> need require

Reply:
It'll actually do either require or import depending on your output format.  I'll add a note to that effect.

Reply:
But the syntax in the *grammar* is always `import`.

Reply:
No, I only meant this doesn't sound grammatical. At least I don't understand what's written there even after the explanation.

Reply:
In the .peggy file, you do this:

`import {foo} from "bar"`

In the generated .js file, you get this:

`var peg$import0 = require("bar")`

or

`import * as peg$import0 from "bar"`

depending on if you specify `--format commonjs` or `--format es`.

I'd love suggestions for wording, please.

Reply:
What about: 

```
  BREAKING: Allow imports
  from external compiled grammars inside a source grammar, using `import
  {rule} from "external.js"`.  Note that this syntax will generate either
  `import` or `require` in the JavaScript output, depending on the value of
  the `format` parameter.  This will need require explicit support from
  plugins, with a few new AST node types and a few visitor changes.
```

Reply:
I found the typo you were talking about.  Hopefully the edit above is more legible.

Reply:
Yep, that's it.